src/components/program/ProgramEndorsements.jsx

```diff
@@ -0,0 +1,52 @@
+import { useContext } from 'react';
+import { faQuoteLeft } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ProgramContext } from './ProgramContextProvider';
+
+const ProgramEndorsements = () => {
+  const { program: { corporateEndorsements } } = useContext(ProgramContext);
+
+  return corporateEndorsements?.length > 0
+    ? (
+      <div className="endorsements p-2">
+        <h2 className="program-section-heading">Program endorsements</h2>
+        <div className="quote-icon-wrapper">
+          <FontAwesomeIcon icon={faQuoteLeft} className="quote-icon" size="2x" />
+        </div>
+        <div className="row">
+          {
+            corporateEndorsements.map(({
+              corporationName: corporation,
+              image,
+              individualEndorsements: endorsements,
+            }) => {
+              const { endorser: { givenName, familyName, position }, quote } = endorsements[0];
+              const title = position ? position.title : '';
+              const author = `${givenName} ${familyName}`;
```

Comment:
one way could be to move these const statement out of the `return` and convert them into one liner `arrow` functions like

```javascript
const formatAuthorFullName = endorser => `${endorser.givenName} ${endorser.familyName}`;
const title = endorser => endorser.position ? endorser.position.title : '';
```